src/client/automation/playback/type/index.js

```diff
@@ -25,7 +25,9 @@ var SPECIAL_KEYS    = testCafeCore.KEY_MAPS.specialKeys;
 export default class TypeAutomation {
     constructor (element, text, typeOptions) {
         this.element = TypeAutomation.findTextEditableChild(element) || element;
-        this.text    = text.toString();
+
+        // eslint-disable-next-line no-restricted-properties
+        this.text = text.toString();
```

Comment:
We can avoid so many comments for linter in this file. Please rename `this.text` -> `this.typingText`